src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -532,11 +532,23 @@ def _create_pgm_input_transformers(self):
         parallel = self._get_pp_attr("trafo", "parallel", 1)
         sn_mva = self._get_pp_attr("trafo", "sn_mva")
         switch_states = self.get_switch_states("trafo")
-        winding_types = self.get_trafo_winding_types()
 
-        tap_side = self._get_pp_attr("trafo", "tap_side")
+        tap_side = self._get_pp_attr("trafo", "tap_side", np.nan)
```

Comment:
`tap_side` in `pandapower` is a string right? Why do you use the default value `np.nan` instead of `None`?

Reply:
Yes, its a string. Changed default to None